src/server/server.test.ts

```diff
@@ -34,7 +34,7 @@ describe('server test suite', () => {
     const port = server.http.address().port;
     await request(`http://localhost:${port}`).get('/').expect(200);
     await server.stop();
-    expect(await request(`http://localhost:${port}`).get('/').timeout(500)).toBeUndefined();
+    return expect(request(`http://localhost:${port}`).get('/').timeout(500)).rejects.toThrow(/ECONNREFUSED/);
```

Comment:
does toThrow actually return a promise? this looks slightly odd but I'm a bit green with this library

Reply:
It would appear so. I had real good time trying to solve that test.

Reply:
ok cool